app/src/main/java/edu/uc/willi6jd/mycaloriecounter/FoodActivity.kt

```diff
@@ -91,7 +91,7 @@ private class foodCustomerAdapter(context: Context): BaseAdapter(){
             val calTextView = rowMain.findViewById<TextView>(R.id.listFoodCal)
             calTextView.text = cal.get(position)
 
-            val fatTextView = rowMain.findViewById<TextView>(R.id.listFoodFat)
```

Comment:
"TextView" is the majority of this variable name, and that's also the type, which is redundant.  Make the variable name descriptive of its contents.